blocks/event-agenda-component/event-agenda-component.js

```diff
@@ -0,0 +1,127 @@
+import { getLibs } from '../../scripts/utils.js';
+import { getIcon, handlize, generateToolTip } from '../../utils/utils.js';
+
+const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+const miloLibs = getLibs();
+
+await Promise.all([
```

Comment:
Let's move these + the miloLibs = getLibs() into the init call. The convention is that these. I don't think this will break anything but we should try to load the modules with as little side effects as possible.